L/Libical/build_tarballs.jl

```diff
@@ -0,0 +1,83 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Libical"
+version = v"3.0.9"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/libical/libical/releases/download/v$(version)/libical-$(version).tar.gz",
+                  "bd26d98b7fcb2eb0cd5461747bbb02024ebe38e293ca53a7dfdcb2505265a728"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd libical-*
+
+apk add glib-dev libxml2-dev
+
+# Don't try this at home, it's evil
+ln -s /opt/${host_target}/${host_target}/sys-root/usr/lib/libc.so /usr/lib/libc.so
+
+FLAGS=(
+    -DCMAKE_BUILD_TYPE=Release
+    -DSHARED_ONLY=true
+    -DICAL_BUILD_DOCS=false
+    -DGOBJECT_INTROSPECTION=false
+    -DLIBICAL_BUILD_TESTING=false
+    -DWITH_CXX_BINDINGS=false
+)
+
+# cross compiling libical requires a binary from the native build
+(
+    mkdir native_build && cd native_build
+    # It would be nice to have `HOST_PKG_CONFIG_*` variables.
```

Comment:
I'm not sure how useful it would be, but maybe there could be a way to switch between host mode and target mode or something

Reply:
https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/917#issuecomment-759126993